packages/compass-components/src/components/accordion.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from 'react';
+import { spacing } from '@leafygreen-ui/tokens';
+import { css } from '@emotion/css';
+import Icon from '@leafygreen-ui/icon';
+import { uiColors } from '@leafygreen-ui/palette';
+import { useId } from '@react-aria/utils';
+
+const labelStyles = css({
+  padding: 0,
+  margin: 0,
+  fontWeight: 'bold'
+});
+
+const buttonStyles = css({
+  display: 'flex',
+  alignItems: 'center',     
+  borderStyle: 'solid',
+  borderColor: 'transparent',
+  borderRadius: '6px',
+  borderWidth: '1px',
+  background: 'none',
+  '&:focus-visible': {
+    borderColor: uiColors.green.base
+  }
+})
+const containerStyles = css({
+  marginTop: spacing[3],
+  display: 'flex',
+  alignItems: 'center',
+  '&:hover': {
+    cursor: 'pointer'
+  }
+})
+interface AccordionProps {
+  dataTestId?: string,
+  text: string
+}
+function Accordion(props: React.PropsWithChildren<AccordionProps>) : React.ReactElement {
+  const [open, setOpen] = useState(false)
+  const contentId = useId();
+  return (
+    <div data-testid={props.dataTestId}>
+      <div
+        className={containerStyles}
+        onClick={(e) => setOpen(!open)}
```

Comment:
So this definitely works as expected due to event bubbling, but I really truly believe that a better place for this onClick would be a button below. Also I'm not sure if you see these eslint errors, but because this is a non-interactive element, eslint a11y checks errors out here:

<img width="800" alt="image" src="https://user-images.githubusercontent.com/5036933/141138217-1a60219e-3458-4510-b785-ff0239041f12.png">